src/common/StatementMetadata.cpp

```diff
@@ -167,7 +167,7 @@ const char* StatementMetadata::getPlan(bool detailed)
 
 	if (plan->isEmpty())
 	{
-		UCHAR info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
```

Comment:
It's very bad idea to poison sources here and there with explicit types casts.
That should be solved using correct types, not casts